CHANGELOG.md

```diff
@@ -12,6 +12,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Policy load API endpoints now default to the `application/x-yaml` content-type
   if no content type is provided in the request
   ([conjurinc/dap-support#74](https://github.com/conjurinc/dap-support/issues/74)).
+- The k8s authenticator correctly authenticates an app using the host ID to
+  specify the k8s resource constraints and an annotation to specify the authenticator
+  container name ([cyberark/conjur#1535](https://github.com/cyberark/conjur/issues/1535),
+  [conjurinc/dap-support#79](https://github.com/conjurinc/dap-support/issues/79)).
```

Comment:
I would state that `The k8s authenticator correctly authenticates an app using the host ID to
  specify the k8s resource constraints and an annotation to specify the authenticator
  container name using the "authn-k8s" prefix`.

If the customer will specify the authenticator container name using the `kubernetes` prefix like this: `kubernetes/authentication-container-name` then it would have worked before as well.

I didn't expect users to use the `authn-k8s` prefix unless they are defining the application identity in annotations but this fix is in place as we can't assume that.

Reply:
Sure, I'll make that change. 

> I didn't expect users to use the authn-k8s prefix unless they are defining the application identity in annotations but this fix is in place as we can't assume that.

Yes, this comes from a support case where this very scenario occurred :)